src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -419,7 +412,8 @@ private String retrieveField(XPath xpath, String expression, Document eomFileDoc
         return getNodeAsString(node);
     }
 
-    private String transformField(final String originalFieldAsString,
+    @SuppressWarnings("unchecked")
```

Comment:
I believe this annotation is unnecessary - can you please remove it or show why is it used, because when I removed it locally that didn't generate any warnings.

Reply:
Indeed it generates - the last method parameter final Entry<String, Object>... contextData